go.mod

```diff
@@ -29,3 +29,5 @@ require (
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/cloudwego/frugal => github.com/cloudwego/frugal v0.1.14-0.20240228092322-657cf8689f17
```

Comment:
since it's the same repo, you can just use `go get github.com/cloudwego/frugal@657cf8689f17` to update `go.mod`, instead of `replace`

Reply:
We can release an official version after testing frugal on both amd64/arm64 and then update it here

Reply:
done!